src/DotLiquid/StandardFilters.cs

```diff
@@ -845,7 +814,7 @@ public static string Default(string input, string @defaultValue)
 
         private static bool IsReal(object o) => o is double || o is float || o is decimal;
 
-        private static object DoMathsOperation(Context context, object input, object operand, Func<Expression, Expression, BinaryExpression> operation)
+        internal static object DoMathsOperation(Context context, object input, object operand, Func<Expression, Expression, BinaryExpression> operation)

```

Comment:
Can we move this to another class, rather than LegacyFilters calling into StandardFilters? Maybe merge `ExpressiongUtility` and this into a `MathsOperation` class?

Reply:
This could be raised as an issue, to be addressed later.

Reply:
I don't want to mess with Math ops at the moment. Agree to defer.